extra_views/contrib/mixins.py

```diff
@@ -3,6 +3,11 @@
 import datetime
 import functools
 import operator
+try:
+    from collections import OrderedDict
+except ImportError:
+    # python 2.6 or earlier, use backport
+    from ordereddict import OrderedDict
```

Comment:
you can fallback to django.utils.datastructures.SortedDict here instead of using a backport